### PR TITLE
conftest and helm cli added

### PR DESCRIPTION
**Pull request by ckavili**

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -4,13 +4,22 @@ RUN INSTALL_PKGS="git" && \
...
     rpm -V $INSTALL_PKGS $UTIL_PKGS && \
     curl -X GET https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64 -L --output /usr/bin/yq && \
     chmod +x /usr/bin/yq && \
     curl -X GET https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_VERSION}/openshift-client-linux-${OC_VERSION}.tar.gz --output /usr/bin/oc.tar.gz && \
     tar xvzf /usr/bin/oc.tar.gz -C /usr/bin/ && \
     rm -rf /usr/bin/{oc.tar.gz,README.md,kubectl} && \
+    curl -LJO https://github.com/open-policy-agent/conftest/releases/download/v${CONFTEST_VERSION}/conftest_${CONFTEST_VERSION}_Linux_x86_64.tar.gz && \
+    tar xvzf conftest_${CONFTEST_VERSION}_Linux_x86_64.tar.gz -C /usr/bin/ && \
+    rm -rf conftest_${CONFTEST_VERSION}_Linux_x86_64.tar.gz && \
+    curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
```
Comment (raballew) [inline Containerfile:19]: Can we use https://mirror.openshift.com/pub/openshift-v4/clients/ instead?

**Review by raballew — CHANGES_REQUESTED** [review]
I would like to use the OpenShift mirror as much as possible.

File: Containerfile (hunk 1 above)
Comment (ckavili) [reply Containerfile:19]: of course we can :)

**Review by raballew — APPROVED** [review]
/lgtm

**Pull request merged**